docs/sources/features/datasources/loki.md

```diff
@@ -71,7 +71,7 @@ Once the result is returned, the log panel shows a list of log rows and a bar ch
 
 <div class="medium-6 columns">
   <video width="800" height="500" controls>
-    <source src="https://grafana.com/static/assets/videos/explore_loki.mp4" type="video/mp4">
+    <source src="/assets/videos/explore_loki.mp4" type="video/mp4">
```

Comment:
When I try testing the docs locally (`make docs`), I can't see the video.

Reply:
Docker image needs to be updated.

Reply:
Ah... thanks for clearing that up @robbymilo.